src/GraphQL.ApiTests/net5/GraphQL.approved.txt

```diff
@@ -1232,7 +1232,7 @@ namespace GraphQL.Execution
     {
         System.Collections.Generic.IReadOnlyDictionary<GraphQLParser.AST.GraphQLField, System.Collections.Generic.IDictionary<string, GraphQL.Execution.ArgumentValue>>? ArgumentValues { get; }
         System.Threading.CancellationToken CancellationToken { get; }
-        System.Collections.Generic.IReadOnlyDictionary<GraphQLParser.AST.GraphQLField, System.Collections.Generic.IDictionary<string, GraphQL.Execution.DirectiveInfo>>? DirectiveValues { get; }
+        System.Collections.Generic.IReadOnlyDictionary<GraphQLParser.AST.ASTNode, System.Collections.Generic.IDictionary<string, GraphQL.Execution.DirectiveInfo>>? DirectiveValues { get; }
```

Comment:
`ASTNode` or `IHasDirectivesNode` here?

Reply:
`IHasDirectivesNode` is suitable for those who wants to call `.Directives` on it. I don't think someone wants to do it here. I would use `ASTNode`.It looks more logical here.

Reply:
Makes sense, thanks!